### ISSUE TITLE
Website preview is very poor

### ISSUE THREAD

**Issue by WhyNotHugo** [marktext/website#35]
Since the website **requires** JavaScript (which seems kinda weird), search engine previews just renders this:

![3BF04E7B-F670-4B2C-BF21-935A50D96E8D](https://user-images.githubusercontent.com/730811/96376285-4bc9cd00-117e-11eb-9ea7-b27d7708e137.jpeg)

There should be some better fallback here, since this is what people also see when JS is off, or by certain a11y tools.

**Comment by Jocs** [comment]
Please visit [www.marktext.cc](http://www.marktext.cc/)

**Comment by WhyNotHugo** [comment]
Issue seems to be still there: 

![We're sorry but marktext doesn't work properly without JavaScript enabled. Please enable it to continue.](https://github.com/marktext/website/assets/730811/e2b9da86-543e-4d04-aedb-12c06d12daed)

**Comment by WhyNotHugo** [comment]
> We're sorry but marktext doesn't work properly without JavaScript enabled. Please enable it to continue.

**Comment by WhyNotHugo** [comment]
The preview generated by search engines still doesn't really say anything useful:

![image](https://github.com/marktext/website/assets/730811/abd4c4d4-81a2-4ed9-901b-b4fe62b718ce)

I think that the problem is that there's no text on the page itself; it only loads the webapp which then loads the text.
